src/plugins/pdf-viewer/info.js

```diff
@@ -34,6 +33,6 @@ export default class PdfViewer {
   }
 
   getCdnResources(content) {
-    return content.type === SOURCE_TYPE.internal && content.url ? [content.url] : [];
+    return content.type === SOURCE_TYPE.internal && content.sourceUrl ? [content.sourceUrl] : [];
```

Comment:
Here it's probably `content.sourceType`, no? 🤔

Reply:
Thanks, corrected! :pray: